actors/evm/src/lib.rs

```diff
@@ -340,8 +337,8 @@ impl ActorCode for EvmContractActor {
                 Ok(IpldBlock::serialize_cbor(&ret)?)
             }
             Some(Method::GetBytecodeHash) => {
-                let multihash = Self::bytecode_hash(rt)?;
-                Ok(IpldBlock::serialize_cbor(&multihash)?)
+                let hash = Self::bytecode_hash(rt)?;
+                Ok(IpldBlock::serialize_cbor(&hash)?)
```

Comment:
```suggestion
                log::trace(target: "evm", "GetBytecodeHash: {:#}", hash);
                Ok(IpldBlock::serialize_cbor(&hash)?)
```

Reply:
feel free to ignore this

Reply:
Let's not just drop random log statements all over the place, that'll be a pain to maintain and doesn't add any value.

If we want to log these, we can do it from the dispatch macro (where we can even gate these logs behind the testing feature flag).